Gaming.html

```diff
@@ -55,96 +55,118 @@
 /*			width: 600px;
 			height: 350px;*/
 			margin-top: 1%;
-			/* padding-bottom: 40%; */
-			width: 900px;
-			height: 750px;
+			padding-bottom: 40%;
+			width: 80%;
+			height: 0;
```

Comment:
Are these changes necessary?

Reply:
This might be the conflicts with the newest-version code, so you can just instead them by the newest items, and just use the "Countdown" part.